Documentation/Blazorise.Docs.Server/Startup.cs

```diff
@@ -53,7 +53,10 @@ public void ConfigureServices( IServiceCollection services )
             } )
             .AddBootstrap5Providers()
             .AddFontAwesomeIcons()
-            .AddBlazoriseRichTextEdit()
+            .AddBlazoriseRichTextEdit( options =>
+            {
+                options.UseTables = true;
```

Comment:
Is there any special reason why this isn't on by default?

Reply:
Because it is an unofficial package. I have made the comment in docs so that users ara aware of it. Once that quilljs adds official support for tables then we can enable it by default.